src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.NormalPriorityProcessor.cs

```diff
@@ -232,18 +232,15 @@ private IEnumerable<DocumentId> GetPrioritizedPendingDocuments()
                         {
                             // First the active document
                             var activeDocumentId = this.Processor._documentTracker.GetActiveDocument();
-                            if (activeDocumentId != null && _higherPriorityDocumentsNotProcessed.ContainsKey(activeDocumentId))
+                            if (activeDocumentId != null)
                             {
                                 yield return activeDocumentId;
                             }
 
                             // Now any visible documents
                             foreach (var visibleDocumentId in this.Processor._documentTracker.GetVisibleDocuments())
```

Comment:
how is visibility determined?

Reply:
we have visual studio document tracking service that tracks active file.

Reply:
and workspace tracks what files are opened in host.

Reply:
so "Visibility = Open"?  Should this be called GetOpenDocuments then?

Reply:
so. the tracker service also track visible (editor that is shown to users) as well.

so, priority is 

active file (where focus is on)
visible files (document that is visible to users)
highpirority files (opened files + ones that specifically asked by users)